### PR TITLE
PR to fulfill pythons style guide rules

### PR DESCRIPTION
**Pull request by joachimwolff**

### CODE REVIEW

File: hicexplorer/reduceMatrix.py [hunk 1]
```diff
@@ -153,17 +156,19 @@ def reduce_matrix(matrix, bins_to_merge, use_triu=True, diagonal=False):
...
     # each original col and row index is converted
     # to a new index based on the bins_to_merge.
     # For example, if rows 1 and 10 are to be merged
     # then all rows whose value is 1 or 10 are given
     # as new value the index in the bins_to_merge list.
 
-    map_ = np.zeros(ma.shape[0], dtype=int) - 1     # -1 such that all cases not replaced by the next loop
-                                                    # can be identified later. Those cases that remain as -1
-                                                    # are for the rows/cols not appearing in the bins_to_merge
+    # -1 such that all cases not replaced by the next loop
```
Comment (bgruening) [inline hicexplorer/reduceMatrix.py]: You spitted the comments here.

File: hicexplorer/hicSumMatrices.py [hunk 2]
```diff
@@ -1,4 +1,5 @@
-import sys, argparse
+# import sys
```
Comment (bgruening) [inline hicexplorer/hicSumMatrices.py]: If all tests succeed, please remove these imports.

File: hicexplorer/HiCMatrix.py [hunk 3]
```diff
@@ -74,7 +75,8 @@ def __init__(self, matrixFile=None, file_format=None, skiprows=None, chrnameList
                 self.maskBins(np.flatnonzero(row_sum==0))
                 """
             elif file_format == 'lieberman':  # lieberman format needs additional arguments : chrnameList
-                lieberman_data = self.getLiebermanBins(filenameList = matrixFile, chrnameList = chrnameList)
+                lieberman_data = self.getLiebermanBins(
```
Comment (bgruening) [inline hicexplorer/HiCMatrix.py]: This does not need to be broken, we do not follow the 80chars recommendation.

File: hicexplorer/HiCMatrix.py [hunk 4]
```diff
@@ -103,11 +104,14 @@ def load_h5(matrix_filename):
             # get intervals
             intvals = {}
             for interval_part in ('chr_list', 'start_list', 'end_list', 'extra_list'):
-                intvals[interval_part] = getattr(f.root.intervals, interval_part).read()
-            cut_intervals = zip(intvals['chr_list'], intvals['start_list'], intvals['end_list'], intvals['extra_list'])
+                intvals[interval_part] = getattr(
+                    f.root.intervals, interval_part).read()
```
Comment (bgruening) [inline hicexplorer/HiCMatrix.py]: Same here please revert this.

File: hicexplorer/HiCMatrix.py [hunk 5]
```diff
@@ -381,8 +388,10 @@ def getRegionBinRange(self, chrname, startpos, endpos):
             exit()
 
         try:
-            startbin = sorted(self.interval_trees[chrname][startpos:startpos + 1])[0].data
-            endbin = sorted(self.interval_trees[chrname][endpos:endpos + 1])[0].data
+            startbin = sorted(
+                self.interval_trees[chrname][startpos:startpos + 1])[0].data
```
Comment (bgruening) [inline hicexplorer/HiCMatrix.py]: please revert these changes, it makes everything more complicated and its hard to review.

File: hicexplorer/HiCMatrix.py [hunk 6]
```diff
@@ -664,35 +681,45 @@ def convert_to_obs_exp_matrix(self, maxdepth=None, zscore=False, perchr=False):
...
+                # be closer.
+                diagonal_length = max(
+                    diagonal_length, distance_len[bin_dist_plus_one])
 
                 if diagonal_length == 0:
                     mu[bin_dist_plus_one] = np.nan
                 else:
-                    mu[bin_dist_plus_one] = np.float64(sum_value) / diagonal_length
+                    mu[bin_dist_plus_one] = np.float64(
+                        sum_value) / diagonal_length
```
Comment (bgruening) [inline hicexplorer/HiCMatrix.py]: urgs.

File: hicexplorer/hicBuildMatrix.py [hunk 7]
```diff
@@ -337,21 +337,22 @@ def get_rf_bins(rf_cut_intervals, min_distance=200, max_distance=800):
...
+            new_end.append(end[idx - 1])
         new_chrom.append(chrom[idx])
 
     new_end.append(end[-1])
     assert len(new_chrom) == len(new_start), "error"
     assert len(new_end) == len(new_start), "error"
 
     intervals = zip(new_chrom, new_start, new_end)
-    intervals = [(chrom, start, end) for chrom, start, end in intervals if end - start >= min_distance]
+    intervals = [(chromosome, start_position, end_position) for chromosome, start_position,
```
Comment (bgruening) [inline hicexplorer/hicBuildMatrix.py]: This looks wrong. Wrong merge?

Comment (joachimwolff) [inline hicexplorer/hicBuildMatrix.py]: Flake8 liniting complaint about the redefinition of 'start', 'end' and 'chrom' which were already defined in 313.

**Comment by joachimwolff** [comment]
I have redone it without 80 lines rule. I will create another PR.
